parsec/core/fs/workspacefs/entry_transactions.py

```diff
@@ -425,6 +460,12 @@ async def file_resize(self, path: FsPath, length: int) -> EntryID:
 
             # Perform resize
             await self._manifest_resize(manifest, length)
-
+            self._send_event(
+                CoreEvent.FS_ENTRY_UPDATED,
+                id=manifest.id,
+                reason=FSEntryUpdatedReason.FILE_RESIZE,
+                entry_id=manifest.id,
+                entry_name=path.name,
```

Comment:
`FSEntryUpdatedReason.FILE_RESIZE` is already triggered in `file_transaction.py` but without the `entry_name` field.
We'd better just remove `entry_id`/`entry_name` here to avoid such inconsistency ;-)

and as a matter of fact it seems you fixed a bug here given the event was missing here \o/